OSDOCS-4265: Updates some ROSA cli items for CloudFront (#66471)
openshift/openshift-docs: snippets/oidc-cloudfront.adoc

//This snippet appears in the following assemblies:
//
// * ../rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc

:_content-type: SNIPPET
[NOTE]
====
ROSA CLI 1.2.7 introduces changes to the OIDC provider endpoint URL format for new clusters. {product-title} cluster OIDC provider URLs are no longer regional. The AWS CloudFront implementation provides improved access speed and resiliency and reduces latency.

> Because this change is only available to new clusters created by using ROSA CLI 1.2.7 or later, there are no supported migration paths for existing OIDC-provider configurations.

Reviewer: ```suggestion
Because this change is only available to new clusters created by using ROSA CLI 1.2.7 or later, existing OIDC-provider configurations do not have any supported migration paths.
```
Per the ISG, rewrite to avoid "there is" at the beginning of an independent clause: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#t. My edit is one possibility.
